airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/context.py

```diff
@@ -222,21 +209,15 @@ def docker_image(self) -> str:
         return f"{self.docker_repository}:{self.docker_image_tag}"
 
     @property
-    def docker_hub_username_secret(self) -> Optional[Secret]:
-        if self.docker_hub_username is None:
-            return None
-        return self.dagger_client.set_secret("docker_hub_username", self.docker_hub_username)
+    def local_secret_store_name(self) -> str:
+        return f"{self.connector.technical_name}-local"
 
     @property
-    def docker_hub_password_secret(self) -> Optional[Secret]:
-        if self.docker_hub_password is None:
-            return None
-        return self.dagger_client.set_secret("docker_hub_password", self.docker_hub_password)
-
-    async def get_connector_secrets(self) -> Dict[str, Secret]:
-        if self._connector_secrets is None:
-            self._connector_secrets = await secrets.get_connector_secrets(self)
-        return self._connector_secrets
+    def local_secret_store(self) -> Optional[LocalDirectorySecretStore]:
```

Comment:
It feels a little random to stash the local secrets info here, when we're getting it from helper functions in `python_connectors.py` and `java_connectors.py`. Would it be possible to follow the same pattern you used for the other secret stores?

Reply:
I tied the `local_secret_store` to the connector context because:
* We have one context instance per connector under test
* We want a local secret store per connector (the content of the `secrets` folder)